examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -70,7 +70,7 @@
     "\n",
```

Comment:
<p>mm.MLPBlock(</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[128,256],</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;activation='relu',</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;no_activation_last_layer=True,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dropout=DROPOUT,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;)</p><p> </p><p>should the 256 not be d_model as well?</p><p> </p><p>My suggestion was that it is easier to see that the MLPBlock last hidden dimension and Transformer dimension have to align</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>

Reply:
<p>ok. I can modify that.</p>